src/openrct2/ride/coaster/AlpineCoaster.cpp

```diff
@@ -0,0 +1,7296 @@
+#pragma region Copyright(c) 2014 - 2017 OpenRCT2 Developers
```

Comment:
```suggestion
#pragma region Copyright(c) 2014 - 2022 OpenRCT2 Developers
```

I noticed you marked Oli's conversation as resolved, but didn't update the copyright to 2022, I don't know if this was intentional or not.

Reply:
I did update it?

Reply:
Ah, it has now!

Reply:
Indeed, the full diff shows 2022 instead of 2017. You may have looked at a specific commit, @karst.

Reply:
Yea I was just a bit too slow haha, got fixed a minute earlier in fcd9b73